test/numbers-tests.js

```diff
@@ -1,11 +1,17 @@
 const assert = require('assert');
 const tasks = require('../src/numbers-tasks');
 it.optional = require('../extensions/it-optional');
+const forbidden = require('../extensions/forbidden');
 
 describe('core-js-numbers', () => {
   it.optional('getRectangleArea should return a square of rectangle', () => {
     assert.equal(tasks.getRectangleArea(5, 10), 50);
     assert.equal(tasks.getRectangleArea(5, 5), 25);
+    assert.equal(
+      forbidden.isCommented(tasks.getRectangleArea),
+      false,
+      `Be sure to remove comments from the final solution`
```

Comment:
Why you use backticks if there are no interpolation of variables inside them?

Reply:
Thank you! Corrected.